sys/sys/tree.h

```diff
@@ -348,8 +349,8 @@ struct __no_subobject_bounds {						\
 #define RB_EMPTY(head)			(RB_ROOT(head) == NULL)
 
 #define RB_SET_PARENT(dst, src, field) do {				\
```

Comment:
I'm surprised this hasn't caused issues in userspace.

Reply:
Yeah, not sure how it ended up working, probably it's not used much?

Reply:
Surprised we don't see sys/sys/rb_test failing...